src/Assignments/Monsters/students/11th/Kangyejiii/Monsters.js

```diff
@@ -30,15 +30,38 @@ class Monsters extends Component {
 
   // SearchBox 에 props로 넘겨줄 handleChange 메소드 정의
 
+  componentDidMount() {
+    fetch("https://jsonplaceholder.typicode.com/users")
+    .then(res=>res.json())
+    .then(res=>this.setState({monsters : res}));
+  }
+
+  handleChange = (e) => {
+    this.setState({userInput: e.target.value});
+    }
+
+    handleSearch = () => {
+      let result=this.state.monsters.filter((el)=>(el.name.toLowerCase().includes(this.state.userInput.toLowerCase())));
+      
+      if(!this.state.userInput.trim("")||result.length===0) {
+        return false;
+      } else {
+        return result;
+      } 
+    }
+
   render() {
+    const result = this.handleSearch();
     return (
       <div className="Monsters">
         <h1>컴포넌트 재사용 연습!</h1>
-        {/* <SearchBox handleChange=정의한메소드 /> */}
-        {/* <CardList monsters=몬스터리스트 /> */}
-      </div>
+        <SearchBox handleChange={this.handleChange}/>
+        {result?  <CardList  monsters={result}/> : <div><h2>검색해보세요:-D</h2></div> }
```

Comment:
삼항연산자 와우! 🥳 🥳 🥳 

이 부분도 코드스타일 조금만... 지적하겠습니다 😊
```suggestion
        {result ? <CardList monsters={result} /> : (
          <div>
            <h2>검색해보세요:-D</h2>
          </div>
        }
```